hw/ip/otbn/data/otbn_sec_cm_testplan.hjson

```diff
@@ -201,5 +201,53 @@
       milestone: V2S
       tests: []
     }
+    {
+      name: sec_cm_data_mem_sec_wipe
+      desc: "Verify the countermeasure(s) OTBN.DMEM.SEC_WIPE."
+      milestone: V2S
+      tests: []
+    }
+    {
+      name: sec_cm_instruction_mem_sec_wipe
+      desc: "Verify the countermeasure(s) IMEM.SEC_WIPE."
+      milestone: V2S
+      tests: []
+    }
+    {
+      name: sec_cm_data_reg_sw_sec_wipe
+      desc: "Verify the countermeasure(s) DATA_REG_SW.SEC_WIPE."
+      milestone: V2S
+      tests: []
+    }
+    {
+      name: sec_cm_write_mem_integrity
+      desc: "Verify the countermeasure(s) WRITE.MEM.INTEGRITY."
+      milestone: V2S
+      tests: []
+    }
+    {
+      name: sec_cm_ctrl_flow_count
+      desc: "Verify the countermeasure(s) CTRL_FLOW.COUNT."
+      milestone: V2S
+      tests: []
+    }
+    {
+      name: sec_cm_ctrl_flow_sca
+      desc: "Verify the countermeasure(s) CTRL_FLOW.SCA."
```

Comment:
@prajwalaputtappa: Note this will require a new test. We can just do something simple like execute two programs one has a taken branch the other has an untaken branch and both execute the same number of instructions. Cycle to execute both should be measured and they must be the same.

Reply:
Thanks, @GregAC. I will add this testcase as well.